pkg/rpaas/client/types/types.go

```diff
@@ -14,3 +37,21 @@ type Plan struct {
 	Description string `json:"description"`
 	Default     bool   `json:"default"`
 }
+
+type InstanceAddress struct {
+	Hostname string `json:"hostname,omitempty"`
+	IP       string `json:"ip,omitempty"`
+}
+
+type InstanceInfo struct {
+	Addresses   []InstanceAddress `json:"address,omitempty"`
```

Comment:
Please, make sure the field name accords to JSON tags.

```suggestion
	Addresses   []InstanceAddress `json:"addresses,omitempty"`
```